packages/core/src/components/navbar/navbarDivider.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+// allow the empty interface so we can label it clearly in the docs
+// tslint:disable-next-line:no-empty-interface
+export interface INavbarDividerProps extends IProps {
```

Comment:
you are spreading `...htmlProps` below but not extending from `React.HTMLProps<HTMLDivElement>`. pick one

Reply:
Will fix, but I'll note that this is something a unit test would have caught.

Reply:
sure, but I caught it in CR :)